api/libs/external_api.py

```diff
@@ -107,5 +108,17 @@ def handle_general_exception(e: Exception):
 
 class ExternalApi(Api):
     def __init__(self, *args, **kwargs):
```

Comment:
Let's change this line to 

```python
def __init__(self, app: Blueprint | Flask, *args, **kwargs)
```

Reply:
Sure. That's my previous available version for assigning the app instance. Adapted.